tests/test_cyto_utils/test_load.py

```diff
@@ -254,5 +283,45 @@ def test_load_profiles_file_path_input():
 
     # Testing non-existing file paths should result in expected behavior
     data_file_not_exist: pathlib.Path = pathlib.Path(tmpdir, "file_not_exist.csv")
-    with pytest.raises(FileNotFoundError, match="No such file or directory"):
+    with pytest.raises(FileNotFoundError, match="didn't find the path"):
         load_profiles(data_file_not_exist)
+
+
+def test_is_anndata(tmp_path: pathlib.Path) -> None:
```

Comment:
This is an incredibly elegant test, kudos! 🎉

Reply:
I’d suggest doing `@pytest.mark.parametrize`, that way if a change breaks a subset of these test cases, you’ll see exactly which ones.